spyder/app/mainwindow.py

```diff
@@ -3223,9 +3223,9 @@ def open_switcher(self, symbol=False):
             self.switcher.setup()
         self.switcher.show()
 
-        # Note: The +1 pixel on the top makes it look better
+        # Note: The +6 pixel on the top makes it look better

```

Comment:
Probably +1 pixel is needed in macOS and +6 in Linux/Windows.

@goanpeca, could you check this?

Reply:
Looks good!

<img width="651" alt="Screen Shot 2019-09-11 at 17 39 54" src="https://user-images.githubusercontent.com/3627835/64740508-34b8a600-d4bb-11e9-9f53-bcef488af92c.png">